src/index.js

```diff
@@ -0,0 +1,87 @@
+import './style.css';
+
+function makeList() {
```

Comment:
- [ ] As specified, kindly ensure the usage of **arrow function** instead of the _function_ keyword. This will keep your  _function_ syntax simpler and cleaner.